apps/fhir/bluebutton/errors.py

```diff
@@ -6,11 +6,12 @@
 #
 # Also used by QPP as documented at:
 # https://confluence.cms.gov/display/QPPGUIDE/RESTful+API+style+guide#RESTfulAPIstyleguide-JSON
-def build_error_response(code, message):
+def build_error_response(code, message, details=None):
```

Comment:
the code from #486 is an alternative for this, it would let you just raise an exception with the proper codes, message, detail.